### ISSUE TITLE
OpenRefine fails to launch after building it on MacOS

### ISSUE THREAD

**Issue by wetneb** [OpenRefine/OpenRefine#4414]
There are reports on our mailing list that building OpenRefine 3.5.1 on MacOS results in a broken app that fails to launch:
https://groups.google.com/g/openrefine-dev/c/i8pnxI5TJIY

This should be investigated.

**Comment by thadguidry** [comment]
@wetneb do we know yet how well our new 3.6 version works on MacOS for launching?

**Comment by wetneb** [comment]
I haven't tested it! But given quite a few MacOS users have already tested our snapshot releases for 3.7, and given that 3.6 is packaged in the same way, I would expect it to work well. So we can probably close this soon.

**Comment by antoine2711** [comment]
Hum. I DO have a problem with the login to Google Sheets data…

![image](https://user-images.githubusercontent.com/12217525/180622346-6350b647-d768-484c-bc3e-58b51a2a1486.png)

But I don't believe it's a KEY issue from packaging. I was able to login a first time. I experience the same problem with v3.6rc1 and v3.6.

@thadguidry & @wetneb: to make things worth, henceforth v3.6, I think we have this problem on ALL our platforms. I tested Windows 10 and Ubuntu 22.04 and I see the same thing, an infinite spin. :-(

Regards, Antoine
